app/views/arts/_new_form.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= form_for(@art) do |f| %>
+  <div class="field">
+    <%= f.label "Name:" %>
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label "Artist:"%>
+    <%= f.text_field :artist %>
+  </div>
+
+  <div class="field">
+    <%= f.label "Location:" %>
+    <%= f.text_field :location %>
+  </div>
+
+  <div class="field">
+    <%= f.label "Category:" %>
+    <%= f.text_field :category %>
+  </div>
+
+  <div class="field">
+    <%= f.label "Description:" %>
+    <%= f.text_field :description %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Add New Art", class: 'button small round form_button' %>
+  </div>
+<% end %>
```

Comment:
These two forms are pretty much exactly the same - can you combine them into the same partial? If you just do `<%= f.submit, class: "..." %>` without passing in the name of the submit button, it should automatically generate a pretty okay name for the button based on whether you're updating or creating.